heartbeat/gcp-pd-move.in

```diff
@@ -102,6 +102,11 @@ correctly.
 <shortdesc lang="en">Optional device name</shortdesc>
 <content type="boolean" default="" />
 </parameter>
+<parameter name="stackdriver_logging" unique="1" required="0">
+<longdesc lang="en">Use stackdriver_logging output to global resource (yes, true, enabled)</longdesc>
+<shortdesc lang="en">Use stackdriver_logging</shortdesc>
+<content type="string" default="yes" />
```

Comment:
You should set it to no or false to avoid issues for users already using it (if they dont have the libraries installed).

For it to default to yes you'll have to add it to PARAMETERS, and change the try/except to `pass` for the except part.

Reply:
Thank you for reply.
I fixed default value 'no'.
Thank you!